packages/SwingSet/src/kernel/dynamicVat.js

```diff
@@ -27,23 +27,37 @@ export function makeDynamicVatCreator(stuff) {
    * defines the vat. This should be generated by calling bundle-source on a
    * module whose default export is makeRootObject(), which takes E as a
    * parameter and returns a root object.
+   * @param options an options bundle. The only option defined so far is
+   * 'metered', which defaults to 'true'. If 'true', the new dynamic vat is
+   * subject to a meter that limits the amount of computation and allocation
+   * that can occur during any given crank. Stack frames are limited as well.
+   * The meter is refilled between cranks, but if the meter ever underflows,
+   * the vat is terminated. If 'false', the vat is unmetered.
```

Comment:
Presumably if the creating vat is itself metered it won't have the power to create new, unmetered vats, so this option is subject to being ignored.   If that understanding is correct, then probably you should expand this comment to talk about that.

Or did we decide that a dynamic vat creator could always create an unmetered vat if it chose to and thus the control to be exercised was whether a vat was vested with this power at all?  In that case the "you can create a vat but it has to be metered" case would be realized by logic inside the vat admin vat.  If so, then probably _that_ should be documented.  However, it doesn't look to me like the vat admin vat code that's part of this PR implements such a control mechanism.

Reply:
Yeah, I'm going with the second option for now (anyone who can create a dynamic vat can also create a metered one, whether they're metered or not). I don't think we have a use-case for dynamic vats creating dynamic vats yet, so I don't think we need to build a facet that enforces a `metered: true` option yet. But you're right, I should document that to make it clear.